Update fr-fr.h

lilrawry opened the pull request:
correcting some mistakes & the to do section translated.

lilrawry committed:
Update fr-fr.h

goatshriek commented:
Thanks for putting these updates in!

The static analysis is failing because it has detected a translation but still sees the `todo translate` comment on line 152. If you remove that comment from the string you've translated, it will be good to go.

Also consider translating the other string at line 119 (`L10N_INVALID_STATE_DURING_UTF8_PARSING`), which is the only other French string missing. If you decide to translate this as well, remember to remove the matching translate todo comment on line 117 as well.

lilrawry commented:
well received with thanks, i'll do that asap.

lilrawry committed:
Update fr-fr.h

the `//to do` at line 119 deleted also line 117 has been translated with success

goatshriek commented on include/private/config/locale/fr-fr.h:
```diff
@@ -151,7 +150,7 @@
 
 // todo translate
```
This todo line needs to be removed now that you've added a translation.

lilrawry replied:
all good 👍🏼

goatshriek reviewed (changes requested):
Almost there! Just one more todo comment to remove and it will be good to go.

lilrawry committed:
Update fr-fr.h

line 151 to do line, deleted.

lilrawry commented:
Hi again,
need anything else let me know 
keep me updated 
have a lovely day.

goatshriek commented:
All merged, and thanks again for the contribution!

lilrawry commented:
My pleasure 🤍✨
